src/components/Shifting/ShiftingBoard.tsx

```diff
@@ -109,7 +109,7 @@ export default function BoardView() {
             value={qParams.patient_name}
             onChange={(e) => updateQuery({ [e.name]: e.value })}
             placeholder={t("search_patient")}
-            className="w-full md:w-60"
+            className="w-full md:w-60 mb-1"
```

Comment:
lets not use margin bottom;

Reply:
I have now removed the "mb-1" for the search bar